waspc/data/Generator/templates/server/src/types/index.ts

```diff
@@ -33,25 +31,26 @@ type AuthenticatedOperation<Entities extends WaspEntity[], Result> = (
 type {= userViewName =} = Omit<{= userEntityName =}, 'password'>
 {=/ isAuthEnabled =}
 
-type Operation<Entities extends WaspEntity[], Result> = (
+type Operation<Entities extends _Entity[], Result> = (
   args: any,
   context: {
-      entities: EntityMap<Entities>,
+    entities: Expand<EntityMap<Entities>>,
```

Comment:
@infomiho I ended up doing this in the end: https://github.com/wasp-lang/wasp/pull/945#discussion_r1084175543

The types look much nicer for the user (they never see the `Expand` type, which was my main concern).

Reply:
What do you think about the old idea of using 
```ts
type ContextWithEntities<Entities extends _Entity[]> = {
  entities: Expand<EntityMap<Entities>>
}
```
and then Operation looks like this
```ts
type Operation<Entities extends _Entity[], Result> = (
  args: any,
  context: ContextWithEntities<Entities>,
) => Promise<Result>
```
and AuthenticatedOperation looks like (notice the Expand around UserView)
```ts
type AuthenticatedOperation<Entities extends _Entity[], Result> =  (
  args: any,
  context: ContextWithEntities<Entities> & { user: Expand<UserView> },
) => Promise<Result>
```

Reply:
Even if we don't go with the `ContextWithEntities` type, I would still recomand wrapping `UserView` with `Expand` because it makes it easier for the user to see the fields.

Reply:
Yes, thanks for reminding me. I had to rearrange the placements of `Expand`, but ended up getting this on hover:
![image](https://user-images.githubusercontent.com/22752307/217791749-878195d0-7db3-4d8d-934b-3b9ab4bb7812.png)
Which I think is ideal.